Externals/FreeSurround/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+set(SRCS
+	source/ChannelMaps.cpp
+	source/KissFFT.cpp
+	source/KissFFTR.cpp
+	source/FreeSurroundDecoder.cpp
+	)
```

Comment:
Indentation is 2 spaces, closing paren should be aligned with set (I know we are inconsistent for that one)